net-print/ink/ink-0.5.3.ebuild

```diff
@@ -10,12 +10,6 @@ HOMEPAGE="http://ink.sourceforge.net/"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
-DEPEND=">net-print/libinklevel-0.8"
+DEPEND="net-print/libinklevel"
 RDEPEND="${DEPEND}"
```

Comment:
Did you test with dash as /bin/sh then?

Reply:
yep, it doesn't give the `shift: can't shift that many` error anymore